includes/class-freemius.php

```diff
@@ -1885,15 +1981,21 @@ private function get_email_sections() {
 		 * @param string $public_key
 		 * @param bool   $is_live
 		 * @param bool   $is_premium
+		 * @param string $type
 		 */
-		function init( $id, $public_key, $is_live = true, $is_premium = true ) {
+		function init( $id, $public_key, $is_live = true, $is_premium = true, $type = Freemius::MODULE_TYPE_PLUGIN ) {
```

Comment:
@fajardoleo I think that we agreed that there's no need to have the type in the init due to the automated type identification.